torchvision/utils.py

```diff
@@ -3,22 +3,25 @@
 import torch
 import math
 import numpy as np
+import warnings
 from PIL import Image, ImageDraw
 from PIL import ImageFont
 
-__all__ = ["make_grid", "save_image", "draw_bounding_boxes"]
 
-irange = range
+__all__ = ["make_grid", "save_image", "draw_bounding_boxes", "get_range"]
```

Comment:
I'm not sure we need to expose the `get_range` to other modules because it looks like it's an internal method. It might be better removing it from this list and even prefixing it with an `_` to indicate it's private.

Reply:
I didn’t understand the role of __all__ before, and I have deleted the get_range.